apiservers/engine/API-COMPATIBILITY.md

```diff
@@ -1,7 +1,73 @@
 # Docker Remote API Compatibility
 VIC attempts to be compatible with the Docker remote API; however, there are some differences.  The API differences are documented below, grouped together by functional areas.
 
-## Container APIs
+# support Matrix
+
+## Container API Support
+
+|API|support?|
+|---|---|
+|List Containers|YES|
+|Create a Container|YES|
+|inspect a container| YES|
+|List processes running inside a container|YES|
+|Get Container Logs|YES|
+|Inspect changes on a containers filesystem|YES|
+|Export a container|YES - In the future|
+|Get Container stats based on resource usage|__NO__|
```

Comment:
I would say this is future and not No.  I think from a production standpoint, it's important.